.github/workflows/ci.yml

```diff
@@ -10,6 +10,20 @@ jobs:
     - name: check
       run: git diff-index --check --cached 4b825dc642cb6eb9a060e54bf8d69288fbee4904
 
+  gcc14-x86_64:
+    runs-on: ubuntu-24.04
```

Comment:
Why explicit `ubuntu-24.04` for gcc14 and recent clang vs. `ubuntu-latest` for older entries? Is this difference intentional, or should the new entries maybe also use simply `ubuntu-latest`?

Reply:
Sorry I didn't read the full commit message before I reviewed. I see you had documented this aspect:

```
These jobs explicitly use ubuntu-24.04 because ubuntu-latest still
points to ubuntu-22.04 where these compilers are not available.
```

It's just right, then.